ui/public/i18n/en/translation.json

```diff
@@ -51,7 +52,10 @@
     "timezone_tooltip": "Set the default timezone for current NethVoice installation",
     "create_nethvoice_adm": "Create NethVoice Administrator for users management",
     "remove_nethvoice_adm": "Remove NethVoice Administrator for users management",
-    "set_nethvoice_adm_password": "Set NethVoice Administrator password for users management"
+    "set_nethvoice_adm_password": "Set NethVoice Administrator password for users management",
+    "Error message hostname": "By changing this field all user configurations will be reset.",
```

Comment:
If the string "Error message hostname" is not returned from the backend, let's use something like "error_message_hostname"